### ISSUE TITLE
Incorrect certificate expiration time

### ISSUE THREAD

**Issue by niusmallnan** [niusmallnan/kube-explorer#17]
Kube-explorer: **v0.2.9(UI v2.6.5-kube-explorer-ui-rc1)**

Install K3s, and use kube-explorer to connect it. Take `k3s-serving` as an example, it is a secret of type `kubernetes.io/tls`.

Use openssl to view the expiration time as follows:
```
kubectl --insecure-skip-tls-verify  get secret -n kube-system k3s-serving -o jsonpath='{.data.tls\.crt}' | base64 -d | openssl x509 -noout -text | grep Not
            Not Before: Jul 22 05:40:32 2022 GMT
            Not After : Jul 22 05:40:32 2023 GMT
```

However, the expiry time displayed in the Explorer UI is different.

<img width="650" alt="截屏2022-07-22 下午2 31 03" src="https://user-images.githubusercontent.com/998984/180377455-b708935c-0333-4810-a9fb-5f580a49a95d.png">

**Comment by ly5156** [comment]
@niusmallnan 
1. I tested and verified （`v2.6.7-rc5-kube-explorer-ui-rc1`）that the time displayed on ui is adjusted for the time zone

<img width="1541" alt="image" src="https://user-images.githubusercontent.com/3406205/182353295-6d75271d-49e3-42b3-ae03-1102aee5630f.png">

2. Before v2.6.5, if you adjust the date format, it will cause the date parsing error because of this issue：https://github.com/rancher/dashboard/issues/5979

**Comment by niusmallnan** [comment]
Not sure if this is related to the version of K3s, when this display problem occurs, two certificates are saved in the secret.
This means that content like this can be seen twice. `-----BEGIN CERTIFICATE-----xxx-----END CERTIFICATE-----`

When parsing with openssl, it will only return the first certificate information.
However, parsing in the UI returns the information for the second certificate.

**Comment by niusmallnan** [comment]
Let's track this here. https://github.com/rancher/dashboard/issues/6595

**Comment by ly5156** [comment]
相关联的 issue：https://github.com/rancher/dashboard/issues/7081
对应的 milstone v2.7.2，以及相关 pr 已经合并

**Comment by niusmallnan** [comment]
Fixed by v0.3.2.
